### DISCUSSION TITLE
problems in starting debian-sid

### DISCUSSION THREAD

**Discussion by jorigel** [ltsp/ltsp#766] [Q&A]
Hi,
booting a debian-sid image via ltsp hangs with these massages:
```
loop 0: detected capacity change from 0 to 3123616
mount: mounting /dev/loop 0 on /run/initramfs/ltsp/0/looproot failed: no such device
ltsp command failed: mount -t squashfs -o ro /root/images/debian-sid.img /run/initramfs/ltsp/0/looproot
Stopping plymouth
Aborting ltsp
```
Is there anything to do before building the image or what can I do about it?
thanX

**Comment by alkisg** [comment]
Hi, set DEBUG_SHELL=1 in ltsp.conf to get an initramfs shell, and there try to run the aforementioned `mount -t squashfs ...`command.
If it fails, check why.
Also at that point, what's the output of `ls /root`?

**Comment by jorigel** [comment]
Hi,
I set the debug shell and tried to do the squashfs-mount by hand. Got the answer:
`/sbin/ltsp: eval: line 128: mount -t: not found`

I checked the presence of _/root/images/debian-sid.img_ via ls and of /run/initramfs/ltsp/0/looproot (no contense) and it looks ok to me.

ls /root shows basically the contents of the server's _/srv/ltsp_ - directory as far as the images are listed in the boot-menu.

Additionally I tried to put debian-siduction (a debian sid-based distribution) into an image and I'm sticking at the same point as the debian-sid image. By this I came to the idea that the image must be the reason for this problem and seeking the net for this. I came across some pages telling images sometimes don't have squashfs aktive when built. So I tried to activate it by calling _modprobe squashfs_ in the image before packing it with _ltsp image_. There was no error coming up doing so, but in fact it didn't help.

Do You want me to put one of these test-images into a download path for You? I think the error is reproducible.

**Comment by alkisg** [comment]
The mount command was not found?!

Sure, nice idea, upload the image somewhere and send me a link.
If you prefer to PM the link, use alkisg at gmail.

**Comment by alkisg** [comment]
I tested with a sid chroot.
I got the same problem as you did.
Then I installed the `ltsp` package inside the chroot and everything worked fine.
Please read the http://ltsp.org/docs/installation/ page.

**Comment by jorigel** [comment]
Hi Alkis,
sorry I'm a little late, because it took time to check the images and I ran into additional trouble. I was using snapshots in virtualbox to safe the different steps of building but running into segmented harddiscs this way. 

But afterwards Your tip solved the problem. The debian-sid as well as the siductuion image are now starting.
Like always You're the best ;-)

thank You my dear
